data/transactions/verify/txn_test.go

```diff
@@ -60,25 +61,122 @@ func verifyTxn(s *transactions.SignedTxn, txnIdx int, groupCtx *GroupContext) er
 	return batchVerifier.Verify()
 }
 
+type DummyLedgerForSignature struct {
+}
+
+func (d *DummyLedgerForSignature) BlockHdrCached(basics.Round) (bookkeeping.BlockHeader, error) {
+	return bookkeeping.BlockHeader{
```

Comment:
`bookkeeping.BlockHeader` creation is repeated more than 5 times.
Will be nice to make a function that returns a block header, and use the function everywhere.